.kitchen.yml

```diff
@@ -0,0 +1,23 @@
+---
+driver_plugin: vagrant
+driver_config:
+  require_chef_omnibus: true
+
+platforms:
+  - name: openresty
+    driver_config:
+      box: opscode_ubuntu-14.10_chef-provisionerless.box
+      box_url: http://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-14.10_chef-provisionerless.box
+      memory: 1024
+      natdnshostresolver1: 'on'
+
+suites:
+- name: default
+  run_list:
+  - recipe[openresty::default]
+  encrypted_data_bag_secret_key_path: ~/.chef/nitrous_encrypted_data_bag_key
+  attributes:
+    nitrous:
+      environment: staging
+    datadog:
+      environment: development
```

Comment:
These are environment specific attributes and should not be present in a generic cookbook. Can you please review these settings?